lib/pages/view-post-page.js

```diff
@@ -68,7 +68,7 @@ export default class ViewPostPage extends BaseContainer {
 
 	commentEventuallyShown( comment ) {
 		const self = this;
-		const commentSelector = By.xpath( `//p[text() = "${comment}"]` );
+		const commentSelector = By.css( `p:contains("${comment}")` );
```

Comment:
If you need to work on adding a data-attribute to the wp-calypso interface and would like some help with this please let me know.

You can open a pull request for that here: https://github.com/automattic/wp-calypso